docusaurus.config.js

```diff
@@ -192,11 +192,12 @@ const config = {
         {
           property: "description",
           content:
-            "Jupiter Station is the place for all things related to Jupiter. Here you can access comprehensive documentation, explore statistics and analytics, ecosystem partners that are integrating with Jupiter and read our latest updates.",
+          "Jupiter Metropolis API is the hub for the most powerful set of liquidity APIs in crypto. Unlock seamless access to liquidity and data with Metropolis and find all the API endpoints needed to build a winning business, ranging from Swap API, Token API, Price API, and more.",
```

Comment:
Hmmm, don't think we should change the meta tags for the project for Metropolis. Perhaps if we can override it within then landing page only or if not, some other more drastic solution might be needed